lib/extension_scanner.rb

```diff
@@ -26,20 +26,35 @@ def initialize(target_uri, data_file, opts)
   end
 
   def root_element_xpath
-    '//*[self::extension or self::install]'
+    '//*'
   end
 
-  def create_extension_from_manifest(xml, extension_path, manifest_uri)
-    manifest = Nokogiri::XML(xml)
-    
+  def get_version_from_manifest(manifest)
```

Comment:
Just a note here I wouldn't normally name a method with `get_` as it goes against lint, but this is how upstream named it and I want to avoid merge conflicts in the future.